.github/workflows/create-new-pr.yml

```diff
@@ -0,0 +1,39 @@
+name: new PR greeter
+on:
+  pull_request_target:
+    types: [opened,reopened]
+
+jobs:
+  new_pr_greeter:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout forklift
+        uses: actions/checkout@v3
+
+      - id: codeowner
+        uses: SvanBoxel/codeowners-action@v1
+        with:
+          path: './.github/CODEOWNERS'
+
+      - name: fail if the commenter doesnt exists in codeowners
+        id: allowedaccess
+        run: |
+          echo allowed=$(echo '${{ steps.codeowner.outputs.codeowners }}' | \
+          jq  -r '."tests/" | index("@${{ github.actor }}")') >> $GITHUB_OUTPUT 
```

Comment:
isn't it too strict? I'm ok with letting developers that are not code-owners to initiate the test, how about checking it in a similar way to [how it's done for oVirt's tests](https://github.com/oVirt/ovirt-system-tests/blob/master/.github/workflows/ost.yaml#L37-L38)?

Reply:
refactored